main.tf

```diff
@@ -18,6 +18,25 @@ data "archive_file" "lambda" {
   source_dir  = "${path.module}/lambda"
 }
 
+
+##########################################################################
+#                                                                        #
+#                               KMS Key                                  #
+#                                                                        #
+##########################################################################
+
+resource "aws_kms_key" "marbot" {
+  description         = "KMS CMK for ${var.module_name}"
```

Comment:
why do you think alarms should use an encrypted topic? they don't contain PII data. But we add $1 / month to the bill.

Reply:
@michaelwittig tfsec is flagging this as an error and the flag can be cleared by either ignoring tfsec (not an option in my case), inserting an exception into the code acceptable or adding the encryption

I come from a background where the default behaviour is to add the layer if all things are equal, which in this case I believe them to be. I appreciate all the dollars add up and this is undesirable for sum and happy to put in the ignore statement if you'd prefer.

Reply:
What about a flag to turn encryption on (off by default)?

Reply:
will take a look later today

Reply:
Removed due to time constraints